### ISSUE TITLE
Example idea: tweet queue resource

### ISSUE THREAD

**Issue by eladb** [winglang/wing#360]
We should consider [cdk-tweet-queue] as a nice example of a reusable resource that we can implement in wing. This was an example we used in talks the early days of the CDK.

The referenced repository should be self explanatory.

[cdk-tweet-queue]: https://github.com/cdklabs/cdk-tweet-queue

**Comment by eladb** [comment]
I still think this is a good example to add

**Comment by moralesl** [comment]
I recently took some time to explore more of Wing capabilities and came across this issue. I started implementing it for Twitter, but based on the recent API pricing changes, you [can't poll or search for tweets in the free API tier](https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#item0) anymore.

As I liked the example of interacting with an external API to showcase the capabilites of Wing, I decided to rather go for an stock update poller that retrieves the API key from secrets manager, fetches the freely available [twelve data Time Series API](https://twelvedata.com/docs#time-series), stores the latest price in DynamoDB and send all stock updates to an SQS queue.

It was already implemented and merged as part of https://github.com/winglang/examples/pull/12 so this issue could potentially be closed.
